ivc/src/ivc/columns.rs

```diff
@@ -1,4 +1,14 @@
-use crate::{ivc::interpreter::N_LIMBS_XLARGE, poseidon_8_56_5_3_2::columns::PoseidonColumn};
+use crate::{
+    ivc::interpreter::N_LIMBS_XLARGE,
+    poseidon_8_56_5_3_2::{
+        bn254::{
+            Column as IVCPoseidonColumn, NB_FULL_ROUND as IVC_POSEIDON_NB_FULL_ROUND,
```

Comment:
This approach with renaming I'm somewhat happy about... at least there are no 5 types of `Column` in the same file :)